### PR TITLE
Feature add delete function

### PR DESCRIPTION
**Pull request by KevinTMiller**
Will alphabetize and fix any const issues after merge of 6, 7, 8, and 9 into master

### CODE REVIEW

File: AVNotes/AVNotes.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -159,7 +161,6 @@
 				8713675F1FE4558D00545AAA /* RecordingManager.swift */,
 				870FF26620436BDF00F0B69F /* GradientManager.swift */,
 				87CE568C2049B0E900C5DC03 /* StateManager.swift */,
-				87CE568E2049FD9A00C5DC03 /* StateManagerProtocols.swift */,
```
Comment (KevinTMiller) [inline AVNotes/AVNotes.xcodeproj/project.pbxproj]: I don't want it to delete this..... I'm not sure how to fix

File: AVNotes/AVNotes/Controllers/FileViewController.swift [hunk 2]
```diff
@@ -50,10 +51,10 @@ class FileViewController: UIViewController, UITableViewDelegate, UITableViewData
         fileTableView.dropDelegate = self
         fileTableView.dragInteractionEnabled = true
 
-        NotificationCenter.default.addObserver(self, selector: #selector(updateTableView),
-                                               name: .annotationsDidUpdate, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(updateTableView),
-                                               name: .modelDidUpdate, object: nil)
+//        NotificationCenter.default.addObserver(self, selector: #selector(updateTableView),
```
Comment (willlarche) [inline AVNotes/AVNotes/Controllers/FileViewController.swift]: comments.

File: AVNotes/AVNotes/Extensions/UIViewController+Alerts.swift [hunk 3]
```diff
@@ -32,6 +32,15 @@ extension UIViewController {
         self.present(alertController, animated: true)
     }
 
+    func confirmDestructiveAlert(title: String, message: String, delete: @escaping () -> Void ) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        let deleteAction = UIAlertAction(title: "Delete", style: .destructive) { _ in delete() }
```
Comment (willlarche) [inline AVNotes/AVNotes/Extensions/UIViewController+Alerts.swift]: Consts.

File: AVNotes/AVNotes/Utilities/AudioManager.swift [hunk 4]
```diff
@@ -31,7 +31,10 @@ enum AudioManagerError: Error {
     case errorLoadingFile
 }
 
-class AudioManager: NSObject, AVAudioRecorderDelegate, AVAudioPlayerDelegate {
+class AudioManager: NSObject,
```
Comment (willlarche) [inline AVNotes/AVNotes/Utilities/AudioManager.swift]: Alpha

**Pull request merged**